### DISCUSSION TITLE
Make unencrypted/unauthenticated git:// access fail better/faster -- unfilter port 9418

### DISCUSSION THREAD

**Discussion by SamB** [community/community#40949] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Filtering this port gives confusing results like: 

```
$ git clone git://github.com/factor/factor.git
Cloning into 'factor'...
[ ... Wait for 130 seconds ... ]
fatal: unable to connect to github.com:
github.com[0: 140.82.114.4]: errno=Connection timed out
```

I don't care what you do with the port, so long as when git attempts to connect to it, things go wrong **quickly**, because there are still instructions to use these URLs scattered about, and if you miss the `git://` at the beginning, they look just like the usual URLs.

Probably the easiest thing is to just accept connections and then immediately disconnect, but if you could somehow arrange for the port to be definitively closed that would also work; it looks like it might be a fair amount of work to actually send back an error message first?

Oh, but I guess you had it working before, back when https://github.com/orgs/community/discussions/21987 was posted -- you should turn that back on!

**Comment by ossanna16** [comment]
Hi there @SamB and welcome to our community! Thank you for the feedback!
